curp/src/server/raw_curp/tests.rs

```diff
@@ -36,20 +36,24 @@ impl<C: 'static + Command, RC: RoleChange + 'static> RawCurp<C, RC> {
         let cmd_board = Arc::new(RwLock::new(CommandBoard::new()));
         let spec_pool = Arc::new(Mutex::new(SpeculativePool::new()));
         let uncommitted_pool = Arc::new(Mutex::new(UncommittedPool::new()));
-        let (log_tx, _log_rx) = mpsc::unbounded_channel();
+        let (log_tx, log_rx) = mpsc::unbounded_channel();
+        // prevent the channel from being closed
+        std::mem::forget(log_rx);
         let sync_events = cluster_info
             .peers_id()
             .iter()
             .map(|id| (id.clone(), Arc::new(Event::new())))
             .collect();
+        let mut curp_config = CurpConfig::default();
```

Comment:
Please use `CurpConfigBuilder` to construct the curp config.

Reply:
done